src/fight.cpp

```diff
@@ -685,7 +685,8 @@ void make_corpse(struct char_data * ch)
         extract_obj(o);
       else {
         // Zero out magazines from this and everything it contains.
-        zero_out_magazine_counts(o);
+        // Leave a round in NPC firearms so characters get some indication of what ammo was being used against them
+        zero_out_magazine_counts(o, IS_NPC(ch));
```

Comment:
This works but isn't the easiest to read. Recommend `IS_NPC(ch) ? 1 : 0` so it's clear that the bool is being coerced to an int in the process.